cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -17,7 +17,8 @@ metadata:
 spec:
   replicas: 1
```

Comment:
Since this only has 1 replica, you might consider the Recreate update policy. The default is RollingUpdate.